.gitignore

```diff
@@ -4,3 +4,4 @@ build/
 node_modules/
 styleguide/
 .eslintcache
+dist/
```

Comment:
@cadrimiranda , can you please revert this change?

`dist/` should not be ignored otherwise the compile GitHub action won't work.

Reply:
Sorry, this weekend was a holiday in Brazil and I'm worked almost 24 hours in the last two days. I updated the code and push to git now.